### PR TITLE
 Add 2023.11 release notes to .metainfo.xml

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

File: packaging/app.organicmaps.desktop.metainfo.xml [hunk 1]
```diff
@@ -113,6 +113,36 @@
...
+       <p>Highlights:</p>
+         <ul>
+           <li>New OSM maps data as of November 13, 2023</li>
+           <li></li>
+           <li></li>
+           <li></li>
+         </ul>
+       <p>Search:</p>
+         <ul>
+           <li></li>
```
Comment (rtsisyk) [inline packaging/app.organicmaps.desktop.metainfo.xml:127]: Do you plan to add any text here?

Comment (Ferenc-) [reply packaging/app.organicmaps.desktop.metainfo.xml:127]: @biodranik can you provide the input?

Comment (biodranik) [reply packaging/app.organicmaps.desktop.metainfo.xml:127]: Try to cherry-pick what is relevant from https://etherpad.wikimedia.org/p/OM-release_notes

Comment (Ferenc-) [reply packaging/app.organicmaps.desktop.metainfo.xml]: Done

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**